target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -146,6 +156,22 @@ define Device/netgear_wax620
 endef
 TARGET_DEVICES += netgear_wax620
 
+define Device/netgear_wax630
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Netgear
+	DEVICE_MODEL := WAX630
+	DEVICE_DTS_CONFIG := config@hk01
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	SOC := ipq8074
+	IMAGES := ui-factory.tar factory.ubi sysupgrade.bin
+	IMAGE/ui-factory.tar := append-ubi | wax6xx-netgear-tar
+	DEVICE_PACKAGES += kmod-spi-gpio \
+		ipq-wifi-netgear_wax630
```

Comment:
No need to break the line for just 2 packages

Reply:
Done.